test/LocalizationWebsite/StartupResourcesAtRootFolder.cs

```diff
@@ -26,7 +29,17 @@ public void ConfigureServices(IServiceCollection services)
         {
             loggerFactory.AddConsole(minLevel: LogLevel.Warning);
 
-            app.UseRequestLocalization();
+            var options = new RequestLocalizationOptions();
+            options.SupportedCultures = new List<CultureInfo>()
+            {
+                new CultureInfo("fr-FR")
+            };
+            options.SupportedUICultures = new List<CultureInfo>()
+            {
+                new CultureInfo("fr-FR")
+            };
+
+            app.UseRequestLocalization(options, new RequestCulture(new CultureInfo("en-US")));
```

Comment:
Should use named param for the second parameter here (and elsewhere in other callsites).